src/wp-includes/blocks.php

```diff
@@ -175,11 +175,18 @@ function register_block_script_module_id( $metadata, $field_name, $index = 0 ) {
 	$block_version       = isset( $metadata['version'] ) ? $metadata['version'] : false;
 	$module_version      = isset( $module_asset['version'] ) ? $module_asset['version'] : $block_version;
 
+	$args = array();
+	if ( isset( $metadata['supports']['interactivity'] ) && $metadata['supports']['interactivity'] ) {
```

Comment:
`supports.interactivity` can be `true`, `false` or contain an object. Inside that object, `interactive` can also be `true` or `false`.

So this should be something like this:

```suggestion
	if (
    isset( $metadata['supports']['interactivity'] ) && true === $metadata['supports']['interactivity'] ) ||
    isset( $metadata['supports']['interactivity']['interactive'] ) && true === $metadata['supports']['interactivity']['interactive'] )
  {
```

[This is the schema](https://github.com/WordPress/gutenberg/blob/trunk/schemas/json/block.json#L610-L631).

Reply:
Thank you! Addressed in cc1d909